src/Fantomas.CoreGlobalTool.Tests/CheckTests.fs

```diff
@@ -28,11 +28,17 @@ let ``formatted files should report exit code 0`` () =
     exitCode |> should equal 0
 
 [<Test>]
-let ``invalid files should report exit code 1`` () =
+let ``invalid files should report exit code 1 with check`` () =
```

Comment:
I would remove this `with check` part, it is implied by the file itself.

Reply:
Fine